Contents/mods/Hydrocraft/media/scripts/Carpentry.txt

```diff
@@ -325,7 +317,7 @@ item HCWashtubfull
     ReplaceOnDeplete    =   HCWashtub,
     ReplaceOnUseOn      =   WaterSource-HCWashtubfull,
     Icon            =   HCWashtubfull,
-    
+		DisplayCategory	= CraftCarp,
```

Comment:
craftchem. technically can be a drainable containing water, but main use of this is for biodiesel only